src/app/dts/dts.service.ts

```diff
@@ -58,6 +58,18 @@ export class DtsService {
     return this.http.get(url, {responseType: 'text'});
   }
 
+  /**
+   * Retrieves a rendered template as a certificate populated with data
+   * @param documentType currently limited to 'enrollment'
+   * @param templateKey unique template identifier
+   * @param dataKey unique data identifier
+   */
+  public renderTemplate(documentType: string, templateKey: string, dataKey: string): Observable<string> {
```

Comment:
Just a tip if you ever have more documentTypes than just ` 'enrollment'`, you could create a type definition using union types like so, which is a lot more specific than `string` while still using strings everywhere:
```export type DocumentType = 'enrollment' | 'somethingElse';```

Reply:
Thank you for the tip.  We can apply change this once we're supporting more document types.